### ISSUE TITLE
Drag link to redirect

### ISSUE THREAD

**Issue by Mike-Reddick** [fluks/redirect-link#21]
I'm thinking in something so, add an new column "Drag Link" in this column the user can choose one of 4 or 8 directions, or simple choose the option "none", with this feature the user can redirect link just dragging the link, if you implement 4 directions, for each side the user can set an redirector. If you like this idea, you can fork the open source code from Glitter Drag addon, you can even implement the settings "trigger distance" and "timeout to cancel" from Glitter Drag. I'm giving this idea because I think this is an more easy way to redirect a link than context menu, but is just an idea to you think about.

Thanks so much

**Comment by fluks** [comment]
It would be a quicker way to redirect a link. If the user would enable this feature, we would need to inject content script on every page.

**Comment by Mike-Reddick** [comment]
Yes, is better this be a optional feature, and if you implement it is good also a setting to choose if open in foreground or in background tab, because if redirecting multiple links in the same page, opening in foreground maybe a issue.
